### ISSUE TITLE
Allow to define http header when read file and define collection name when lookup volumes

### ISSUE THREAD

**Issue by gendigbadig** [atroo/node-weedfs#8]
Hi, 

I appreciate your work for creating this module.

But, I need to pass http header when reading file, because the file i stored on seaweed mostly audio file, and i need to send `Range` header for 'duration seek' purposes.
Also, I need to define collection name when lookup volume, so when i know the file-key but collection name is wrong, the volume will not be found.

I have already create PR #7 . Please kindly check my PR 
with my change, i can do this : 
```
// find volume that contains file "21,06c7a1c5333b" with collection named "svara-music"
let opts = {
  collection: "svara-music"
}

seaweedfs.find("21,06c7a1c5333b", opts).then(function(res) {
  console.log(res)
}).catch(function(err) {
  console.log(err)
});

// read file with id "21,06c7a1c5333b" with http-range from bytes 0 to 100000
let opts = {
  headers: {
    'Range': 'bytes=0-100000'
  }
}

seaweedfs.read("21,06c7a1c5333b", opts).then(function(Buffer) {
  console.log(Buffer)
}).catch(function(err) {
  console.log(err)
});
```

Thank you :)

**Comment by phal0r** [comment]
Hi @gendigbadig,

thanks for your PR. I will definitely have a look at it, but I will be busy this week. So you can expect feedback next week. Hope this is ok for you.

Best regards,

Martin

**Comment by gendigbadig** [comment]
Hi @phal0r, 

Thanks for your response. Happy to hear that :)

**Comment by phal0r** [comment]
Hi @gendigbadig ,

I commented directly in the pull request. Only minor changes needed :)

**Comment by gendigbadig** [comment]
Hi @phal0r ,
I can't find your comment in the pull request, can you check it again?

**Comment by phal0r** [comment]
Hey @gendigbadig ,
I did not realize, how the review mode is working, sorry :)

Now you should see the comments, do you?

**Comment by phal0r** [comment]
Since you suggested to use default parameters, we also should update the engine dependency to node >= 6.

**Comment by atroo** [comment]
Available with version 1.4.0
